llvm/test/TableGen/generic-tables.td

```diff
@@ -24,34 +24,35 @@ include "llvm/TableGen/SearchableTable.td"
 
 // CHECK-LABEL: GET_ATable_IMPL
 // CHECK: constexpr AEntry ATable[] = {
-// CHECK-NOT:   { "aaa"
-// CHECK:       { "baz"
-// CHECK:       { "foo"
-// CHECK:       { "foobar"
-// CHECK:       { "bar"
+// CHECK-NOT:   { "aaa", 0x0, 0x0, 0x0 },
+// CHECK:       { "baz", 0x2, 0x6, 0xFFFFFFFF },
+// CHECK:       { "foo", 0x4, 0x4, 0xFFFFFFFF },
+// CHECK:       { "foobar", 0x4, 0x5, 0xFFFFFFFF },
+// CHECK:       { "bar", 0x5, 0x3, 0xFFFFFFFF },
 // CHECK: };
 
 // CHECK: const AEntry *lookupATableByValues(uint8_t Val1, uint16_t Val2) {
 // CHECK:   return &*Idx;
 // CHECK: }
 
-class AEntry<string str, int val1, int val2> {
+class AEntry<string str, int val1, int val2, int val3> {
```

Comment:
Just a thought - wouldn't' the test be more explicit if we would be passing `bits<64> val3`, and instantiate sequences of bits that are longer than 32 bits?

Reply:
Good advice! Done!